src/app/swadhin/layout.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import Navbar from "./components/Navbar";
+
+function HomeLayout({ children }) {
+  return (
+    <>
+      <Navbar />
+      {children}
```

Comment:
why are we using children prop here?

Reply:
This is a layout children is used to render the page here.